app/controllers/courses_controller.rb

```diff
@@ -81,21 +81,22 @@ def create
       if @course.save
         format.html { redirect_to(@course, notice: 'Course was successfully created.') }
       else
-        format.html { render action: "new" , notice: 'Course could not be created.' }
+        format.html { render action: 'new', notice: 'Course could not be created.' }
       end
     end
   end
 
-private
+  private
+
   def course_params
     params.permit(course: [:name, :source_url, :git_branch])
   end
 
   def assign_show_view_vars
-    @exercises = @course.
-      exercises.
-      includes(:course).
-      select {|ex| ex.visible_to?(current_user) }.natsort_by(&:name)
+    @exercises = @course
+                 .exercises
+                 .includes(:course)
+                 .select { |ex| ex.visible_to?(current_user) }.natsort_by(&:name)
```

Comment:
Yes, these need to be fixed manually. What is the preference, aligning the dots vertically or indenting by 2 spaces from the variable?